styles/blocks/footer/footer.less

```
@@ -0,0 +1,95 @@
+.footer {
+
+  background-color: #212122;
+  color: #fff;
+  padding: 30px 0 20px;
+  @media @screen-gt-s {
+    padding-bottom: 30px;
+  }
```

Обычно отделяют пустыми строка блоки стилей
```suggestion

  @media @screen-gt-s {
    padding-bottom: 30px;
  }

```

---

Добавила (и в других файлах тоже).

То есть, корректное форматирование это когда:
1) `.selector` {   если дальше свойства, пустой строки не должно быть
2) если после свойств какой-то вложенный селектор (или блок, или media), то отделить его пустой строкой.

Всё так?

---

да